cli.ts

```diff
@@ -2322,6 +2351,7 @@ yargs(hideBin(process.argv)) // eslint-disable-line no-unused-expressions
             userId_: installIdOrUserId(),
             cliAction: Avo.CliAction.WHOAMI,
             cliInvokedByCi: invokedByCi(),
+            force: undefined,
```

Comment:
is force never sent as some value, or is that just the case in this diff? 🤔

Reply:
It's never sent, just as it was before, and not the Avo TypeScript codegen is making that explicit.